assets/lambda@edge/LambdaOriginRequestIamAuth.test.ts

```diff
@@ -0,0 +1,109 @@
+import { describe, test } from 'node:test';
+import type { CloudFrontRequestEvent, CloudFrontRequestHandler } from 'aws-lambda';
+import { handler } from './LambdaOriginRequestIamAuth';
+
+type CloudFrontRequestContext = Parameters<CloudFrontRequestHandler>[1];
+
+/**
+ * Simple test to debug lambda@edge functions
+ */
+describe('LambdaOriginRequestIamAuth', async () => {
```

Comment:
this doesn't need to be async, you don't need to await test(...)

Reply:
I was using the Node.js builtin test runner where you need to use `async` and `await` but now I see that this project is setup to use jest so I'll use jest and remove async/await